pkg/client/imagestreamsecrets.go

```diff
@@ -0,0 +1,44 @@
+package client
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+// ImageStreamSecretsNamespacer has methods to work with ImageStreamSecret resources in a namespace
+type ImageStreamSecretsNamespacer interface {
+	ImageStreamSecrets(namespace string) ImageStreamSecretInterface
+}
+
+// ImageStreamSecretInterface exposes methods on ImageStreamSecret resources.
+type ImageStreamSecretInterface interface {
```

Comment:
If we want a general `Secrets` interface akin to `Scale`, then I get this sort of interface.  If its specific to `ImageStreams`, I'd prefer to add this to the `ImageStreamInterface` since it logically lives there.

Also, `ImageStreamSecrets(ns).Secrets(name)` that takes a name that isn't a secret name, looks kind of silly.